hbook/h1d.go

```diff
@@ -186,6 +186,64 @@ func (h *H1D) Scale(factor float64) {
 	h.Binning.scaleW(factor)
 }
 
+// AddScaledH1D returns the histogram with the bin-by-bin h1+alpha*h2
+// operation, assuming statistical uncertainties are uncorrelated.
+func AddScaledH1D(h1 *H1D, alpha float64, h2 *H1D) *H1D {
+
+	if h1.Len() != h2.Len() {
+		panic("hbook: h1 and h2 have different number of bins")
+	}
+
+	if h1.XMin() != h2.XMin() {
+		panic("hbook: h1 and h2 have different Xmin")
+	}
+
+	if h1.XMax() != h2.XMax() {
+		panic("hbook: h1 and h2 have different Xmax")
+	}
+
+	hsum := NewH1D(h1.Len(), h1.XMin(), h1.XMax())
+	alpha2 := alpha * alpha
+
+	compute_sum := func(dst, d1, d2 *Dist0D) {
```

Comment:
shouldn't we also fill in the `N` field of `dst`?

Reply:
Of course, sorry I forgot this one!